ci/run_envoy_docker.sh

```diff
@@ -40,9 +40,13 @@ else
   BUILD_DIR_MOUNT_DEST=/build
   SOURCE_DIR="${PWD}"
   SOURCE_DIR_MOUNT_DEST=/source
+  # TODO(phlax): remove jq install once upstream support is added:
+  #    https://github.com/envoyproxy/envoy-build-tools/pull/132
   START_COMMAND=("/bin/bash" "-lc" "groupadd --gid $(id -g) -f envoygroup \
     && useradd -o --uid $(id -u) --gid $(id -g) --no-create-home --home-dir /build envoybuild \
     && usermod -a -G pcap envoybuild \
+    && apt-get update \
```

Comment:
preferably remove this and land https://github.com/envoyproxy/envoy-build-tools/pull/132 before landing this (which will also require some hash updates etc)